models/players.js

```diff
@@ -30,6 +35,20 @@ var returnPlayers = function (players, rb_team_id, res, league){
   }
 }
 
+var formatEvenOdds = function(players){
```

Comment:
I don't know much about mobile vs. web displays, but does this restrict us too much with small screens to force the 2 columns?